test/StandardAssetRegistry.js

```diff
@@ -73,4 +73,97 @@ contract('StandardAssetRegistry', accounts => {
     })
   })
 
+  describe('exists', () => {
+    it('ensures the asset exists after created', async () => {
+      const output = await registry.exists(1)
+      output.should.be.true      
+    })
+    it('ensures does return an assets if it does not exist', async () => {
+      const output = await registry.exists(100)
+      output.should.be.false
+    })
+    it('throws is not valid id', async () => {
+      try {
```

Comment:
Check out the `assertRevert` function as used here: https://github.com/OpenZeppelin/zeppelin-solidity/pull/627/commits/4173800964c5adf65c63339b09f5a3b484ea6507

Reply:
`assertRevert(await registry.exists(true))` instead of the whole trycatch block.